nativelib/src/main/resources/dirent.c

```diff
@@ -37,3 +37,14 @@ int scalanative_readdir(DIR *dirp, struct scalanative_dirent *buf) {
 }
 
 int scalanative_closedir(DIR *dirp) { return closedir(dirp); }
+
+int scalanative_gettempdir(char *buffer, size_t length)
+{
+#ifndef _WIN32
+  const char* tmpdir = "/tmp";
+  strncpy(buffer, tmpdir, length);
+  return 0;
+#else
+  return getWinTempDir(buffer, length);
```

Comment:
You can query on JVM. It should be setup in `java.lang.System` but it hasn't been implemented in native yet.
```
val props = System.getProperties
props.list(System.out)
```
Note that on macOS I get the following so I'm not sure how this is determined:
`java.io.tmpdir=/var/folders/p3/jyf7rkws0899cy450njtg...`